olm-catalog/serverless-operator/metadata/annotations.yaml

```diff
@@ -1,6 +1,6 @@
 annotations:
   operators.operatorframework.io.bundle.channel.default.v1: "4.6"
-  operators.operatorframework.io.bundle.channels.v1: "4.6"
+  operators.operatorframework.io.bundle.channels.v1: 4.6,4.7
```

Comment:
why not keep the quotes, so it's not forgotten when moving to a pure 4.7 channel (so that quotes are needed again to distinguish it from a float) ?

Reply:
This is generated code. When we move to pure 4.7, they'll be readded automatically.